jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -88,27 +88,29 @@ pipeline {
                             echo "buildManifestUrl (x64, tar): ${buildManifestUrl}"
                             echo "artifactUrl (x64, tar): ${artifactUrl}"
 
-                            'integ-test': {
-                                Boolean skipIntegTests = INTEG_TEST_JOB_NAME == ''
-                                echo "${skipIntegTests ? 'Skipping integration tests' : 'Running integration tests'}"
-                                if (!skipIntegTests) {
-                                    def integTestResults =
-                                        build job: INTEG_TEST_JOB_NAME,
-                                        propagate: false,
-                                        wait: true,
-                                        parameters: [
-                                            string(name: 'TEST_MANIFEST', value: TEST_MANIFEST),
-                                            string(name: 'BUILD_MANIFEST_URL', value: buildManifestUrl),
-                                            string(name: 'AGENT_LABEL', value: AGENT_X64)
-                                        ]
+                            parallel([
+                                'integ-test': {
+                                    Boolean skipIntegTests = INTEG_TEST_JOB_NAME == ''
+                                    echo "${skipIntegTests ? 'Skipping integration tests' : 'Running integration tests'}"
+                                    if (!skipIntegTests) {
+                                        def integTestResults =
+                                            build job: INTEG_TEST_JOB_NAME,
+                                            propagate: false,
+                                            wait: true,
+                                            parameters: [
+                                                string(name: 'TEST_MANIFEST', value: TEST_MANIFEST),
+                                                string(name: 'BUILD_MANIFEST_URL', value: buildManifestUrl),
+                                                string(name: 'AGENT_LABEL', value: AGENT_X64)
+                                            ]
 
-                                    env.ARTIFACT_URL_X64_TAR_INTEG_TEST_RESULT = createTestResultsMessage(
-                                        testType: "Integ Tests (x64, tar)",
-                                        status: integTestResults.getResult(),
-                                        absoluteUrl: integTestResults.getAbsoluteUrl()
-                                    )
-                                } 
-                            }
+                                        env.ARTIFACT_URL_X64_TAR_INTEG_TEST_RESULT = createTestResultsMessage(
```

Comment:
Could we remove the uses of bwc and rpm related env variables in the post always blocks?

Reply:
New changes now.